packages/flutter/lib/src/widgets/raw_menu_anchor.dart

```diff
@@ -405,17 +413,60 @@ mixin _RawMenuAnchorBaseMixin<T extends StatefulWidget> on State<T> {
   @protected
   void open({Offset? position});
 
-  /// Close the menu.
+  /// Close the menu and all of its children.
+  ///
+  /// If `inDispose` is true, the menu will close without rebuilding its parent.
+  ///
+  /// Unless the menu needs to be closed immediately, [requestClose] should be
+  /// called instead of [close]. Doing so allows subclasses to control how the
+  /// menu is opened.
   @protected
   void close({bool inDispose = false});
 
+  /// Request that the menu starts opening.
+  ///
+  /// Unless the menu needs to be opened immediately, this method should be
+  /// called instead of [open]. Doing so allows subclasses to control how the
+  /// menu is opened.
```

Comment:
I enforced the idea that this documentation should be written in the perspective of "instructions for future maintainers".
```suggestion
  // Implemented by subclasses to define what to do upon [MenuController.open].
  //
  // This method should not be directly called by subclasses. Its call chain should eventually invoke [open].
```

Reply:
Done